src/components/add-invoice/GenericAddInput.tsx

```diff
@@ -0,0 +1,26 @@
+export const GenericAddInput = () => {
+  return (
+	<>
+			{/* <Controller
+				render={() => (
+					<input
+						className={`w-full focus:outline-none p-1 border-2 bg-none border-[#F1F1F1] rounded-lg ${
```

Comment:
jak robimy prke do mastera to nie moze w niej byc zakomentowanego kodu. Jesli chcesz go sobie zostawic, bo go potrzebujesz, bedziesz przerabial etc to stworz nowego brancha, wrzuc to tam, a z prki usun